code/controllers/subsystem/demo.dm

```diff
@@ -421,7 +421,7 @@ SUBSYSTEM_DEF(demo)
 /datum/controller/subsystem/demo/proc/mark_dirty(atom/movable/M)
 	if(!isobj(M) && !ismob(M))
```

Comment:
I am also curious as to whether in places like this istype(M) or ismovable(M) is not used?

Reply:
because lighting objects are a no-no for now.